bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaWaterSensorHandler.java

```diff
@@ -12,8 +12,8 @@
  */
 package org.openhab.binding.somfytahoma.internal.handler;
 
-import static org.openhab.binding.somfytahoma.internal.SomfyTahomaBindingConstants.CONTACT;
-import static org.openhab.binding.somfytahoma.internal.SomfyTahomaBindingConstants.WATER_DETECTION_STATE;
+import static org.openhab.binding.somfytahoma.internal.SomfyTahomaBindingConstants.*;
+import static org.openhab.binding.somfytahoma.internal.SomfyTahomaBindingConstants.SENSOR_DEFECT_STATE;
```

Comment:
This looks strange. Maybe run the spotless formatter before we merge

Reply:
fixed